app/src/main/java/com/jerboa/ui/components/common/InputFields.kt

```diff
@@ -641,6 +644,50 @@ fun MyMarkdownText(
     )
 }
 
+@Composable
+fun CheckboxField(
+    label: String,
+    checked: Boolean,
+    onCheckedChange: (checked: Boolean) -> Unit,
+) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween,
+        modifier = Modifier.fillMaxWidth(),
+    ) {
+        Text(
+            text = label,
+        )
+        Checkbox(
+            checked = checked,
+            onCheckedChange = onCheckedChange,
+        )
+    }
+}
+
+@Composable
+fun ExpiresField(
+    value: Int?,
+    onIntChange: (Int?) -> Unit,
+    isValid: Boolean,
+) {
+    OutlinedTextField(
+        modifier = Modifier.fillMaxWidth(),
+        value = value?.toString() ?: "",
+        onValueChange = {
```

Comment:
Some weird logic here, but it seems to work. Mostly copied from stackoverflow. The expire days needs positive integers, or if its empty, send null.